terraform/modules/content_delivery_network/main.tf

```diff
@@ -370,6 +370,31 @@ resource "aws_cloudfront_distribution" "tiles" {
     }
   }
 
+  # send all tile cache preview requests to tile cache app
+  ordered_cache_behavior {
+    allowed_methods        = local.methods
+    cached_methods         = local.methods
+    target_origin_id       = "dynamic"
+    compress               = true
+    path_pattern           = "*/preview*"
+    default_ttl            = 300 # 5min for now..switch to 24h
```

Comment:
did you want to switch to 24h before merging this PR?

Reply:
Good catch, I'll switch this back to 24 hrs.